exercises/practice/run-length-encoding/RunLengthEncodingTest.php

```diff
@@ -31,59 +9,154 @@ public static function setUpBeforeClass(): void
         require_once 'RunLengthEncoding.php';
     }
 
+    /**
+     * uuid: ad53b61b-6ffc-422f-81a6-61f7df92a231
+     * @testdox empty string
```

Comment:
The outer description from the `cases` array should be added in front of the test case description. That was one of the lessons I learned from the test generator implementation. So please add them like this:

```suggestion
     * @testdox Run-length encode a string - empty string
```